test/08_api-keys.ts

```diff
@@ -502,6 +510,121 @@ export default () => {
 						expect(apiKey).to.be.a('string');
 						expect(apiKey).to.not.be.empty;
 					});
+
+					it('should not allow requests with a name longer than 50 characters', async function () {
+						const name = 'too-long-name-'.padEnd(51, 'a');
+						const description = 'long description '.padEnd(150, 'a');
```

Comment:
Is the `description` padding here necessary?

Reply:
No, I just liked the word separation :)

Reply:
No, it isn't, since I later added an explicit test case for the max size of both fields :+1: